### PR TITLE
Jim aws setup

### PR DESCRIPTION
**Pull request by jmilleriec**
Changes to common component and inherited components to remove dependency injection issues when going to prod from angular dev

### CODE REVIEW

File: src/app/components/common/common.component.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { ContentAPIService } from '../../services/content.api.service';
+import { Inject } from '@angular/core';
 
 @Component({
```
Comment (columbusdevsayshello) [inline src/app/components/common/common.component.ts:6]: hmm this guy is empty

File: src/app/components/common/common.component.ts [hunk 2]
```diff
@@ -9,24 +10,37 @@ import { ContentAPIService } from '../../services/content.api.service';
...
+  constructor(protected http: HttpClient, 
+    protected contentSrv: ContentAPIService,
+    //protected passedContent: String[]
+    ) {
+    // this.stContent = passedContent;
+    // console.log(passedContent);
   }
 
   ngOnInit() {
     // Get Externalized Content
```
Comment (columbusdevsayshello) [inline src/app/components/common/common.component.ts]: so... ngOnInit basically is empty. @jmilleriec  we should just delete?

File: src/app/components/common/common.component.ts [hunk 3]
```diff
@@ -9,24 +10,37 @@ import { ContentAPIService } from '../../services/content.api.service';
...
+@Injectable()
 export class CommonComponent implements OnInit {
 
-  protected stContent: string[] = [];
+  protected stContent: String[] = [];
   protected rtrnContent: String[][];
-  constructor(protected http: HttpClient, protected contentSrv: ContentAPIService, protected passedContent: string[]) {
-    this.stContent = passedContent;
-    console.log(passedContent);
+  constructor(protected http: HttpClient,
```
Comment (columbusdevsayshello) [inline src/app/components/common/common.component.ts:18]: I don't think this does anything except for the ContentAPIService. So also not needed.

File: src/app/components/signup/signup.component.ts [hunk 4]
```diff
@@ -27,9 +27,15 @@ export class SignupComponent extends CommonComponent{
   constructor(protected http: HttpClient,
     protected contentSrv: ContentAPIService,
     public signupService: SignUpService) {
-    super(http, contentSrv, stContent);
+    super(http, contentSrv);
+    
   }
- 
+  ngOnInit() {
+    super.ngOnInit();
```
Comment (columbusdevsayshello) [inline src/app/components/signup/signup.component.ts]: The super.ngOnInit does nothing here.

File: src/app/components/common/common.component.ts (hunk 1 above)
Comment (jmilleriec) [reply src/app/components/common/common.component.ts:6]: yeah, can get rid of this...

File: src/app/components/common/common.component.ts (hunk 2 above)
Comment (jmilleriec) [reply src/app/components/common/common.component.ts]: we can, the previous challenge was being caused by dependency injection, constructor and ngOnInit, so I took everything out and simply required method invocation on ngOnInit in the inherited class

File: src/app/components/common/common.component.ts (hunk 3 above)
Comment (jmilleriec) [reply src/app/components/common/common.component.ts:18]: we can eliminate

File: src/app/components/signup/signup.component.ts (hunk 4 above)
Comment (jmilleriec) [reply src/app/components/signup/signup.component.ts]: Yeah, i can get rid of that...

**Pull request merged**